src/fiatExchanges/utils.tsx

```diff
@@ -291,3 +291,6 @@ export const filterProvidersByPaymentMethod = (
 ) => {
   return externalProviders?.find((quote) => quote.paymentMethods.includes(paymentMethod))
 }
+
+export const isUserInputCrypto = (flow: CICOFlow, currency: Currency | CiCoCurrency): boolean =>
```

Comment:
I know this isn't new, but I don't actually understand this logic. Why must a user designate an amount in crypto when purchasing Celo? Would be nice if there was a docstring explaining this

Reply:
I don't either :) 

I brought it up to Nitya & Laura. They plan on discussing this week https://valora-app.slack.com/archives/C03D3K379FE/p1668122181856179